src/components/timing/TimingControls.vue

```diff
@@ -42,8 +43,15 @@ const handleStart = async () => {
   loadingStart.value = true
   let start = new Date()
   if (projectField.value !== '') {
-    await dateStorage.postDailyHours( projectField.value, notesField.value )
-    
+    await dateStorage.postDailyHours(
```

Comment:
This code is wrong. postDailyHours recives project name and notes (line 45 was good)

Reply:
This made me see that if we are reciving as a first char a number, capitalizeLetters function (line 193) throws an error. That is because capitalizeLetters expects a string. A number cannot be capitalized. To fix the bug just change de capitalizeLetters.js. Here the code:
`
function capitalizeLetters(string) {
  if (string !== null && string === isNaN) {
    return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
  }
  else {
    return string
  }
}

export default capitalizeLetters
`